### PR TITLE
feat(sigs): Submit proposal for a documentation SIG

### PR DESCRIPTION
**Pull request by brian-armory**
The documentation for Spinnaker can benefit from a group that looks at the documentation set holistically.

### CODE REVIEW

File: sig-documentation/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+### Meeting Cadence
+ - 1 hour every month on Google Hangouts
+ - TBD of every month at TBD PM PST
+
+# Responsibilities
+
+## Description
+
+Documentation SIG is responsible for the direction and organization of the Spinnaker documentation on https://www.spinnaker.io/. The SIG also acts as a place where users and developers can find collaborators and get help creating or modifying documentation.
```
Comment (robzienert) [inline sig-documentation/README.md]: IMO: The documentation SIG should also be taking an active role in proposing and driving initiatives for code-driven doc generation. Take for example our swagger documentation. Take for example our public API or event stream.

File: sig-documentation/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Proposal: Spinnaker Documentation Special Interest Group
+
+## Roles & Responsibilities of the SIG Leads
+
+### SIG Lead(s):
```
Comment (robzienert) [inline sig-documentation/README.md]: cc @ezimanyi -- someone from your side?

Comment (ezimanyi) [reply sig-documentation/README.md]: @dorbin: Is this something you'd be interested in being a part of?

Comment (dorbin) [reply sig-documentation/README.md]: Yes, and in fact I was just in communication with Brian and with Ben Klein about this. I'm glad Brian's taking the lead on this, and I definitely want to be a part of it.

File: sig-documentation/README.md (hunk 1 above)
Comment (brian-armory) [reply sig-documentation/README.md]: I think that's a good idea. @dorbin and @fifthposition, what are your thoughts?

Comment (fifthposition) [reply sig-documentation/README.md]: That sounds right to me.

**Comment by aglover** [comment]
Sounds like @dorbin needs to be added as a SIG lead. Once that's done, I will notify the SC for a vote.

**Comment by aglover** [comment]
LGTM. +1 binding vote.

**Comment by rmeshenberg** [comment]
+1

**Comment by duftler** [comment]
+1

File: sig-documentation/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+### Meeting Cadence
+ - 1 hour every month on Google Hangouts
+ - TBD of every month at TBD PM PST
+
+# Responsibilities
+
+## Description
+
+Documentation SIG is responsible for the direction and organization of the Spinnaker documentation on https://www.spinnaker.io/. The SIG also acts as a place where users and developers can find collaborators and get help creating or modifying documentation.
```
Comment (pstout) [inline sig-documentation/README.md]: The scope of the SIG should be narrowed slightly to exclude the documentation around the governance of the project, which are owned by the steering committee.

File: sig-documentation/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+SIG leads and members will develop, maintain, and prioritize a backlog of documentation projects and requests. These projects and requests are macro-level issues, such as organization. They are independent of releases and features.  This prioritization will be published as the Documentation SIG Roadmap.
+
+## Scope of Work and Goals
+
+* Encourage Spinnaker adoption by creating helpful documentation.
+* Establish a common framework for evaluating and improving documentation.
+* Develop, encourage, and maintain a clearly defined structure and organization for the Spinnaker documentation.
+* Assist developers and SIGs with creating documentation for new features in Spinnaker.
```
Comment (dorbin) [inline sig-documentation/README.md]:
```suggestion
* Help developers and SIGs create documentation for new features in Spinnaker.
```

File: sig-documentation/README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+* Propose and drive initiatives for code-driven doc generation.
+* Facilitate community contributions to the Spinnaker documentation by creating templates and style guides.
```
Comment (dorbin) [inline sig-documentation/README.md]: Would add...
* Maintain and improve the platform for building and publishing spinnaker.io.

**Comment by aglover** [comment]
@demobox what say you?

**Comment by demobox** [comment]
> demobox what say you?

+1 - looks good to me too (binding vote)

**Pull request merged**